frontend/src/components/resource/resources.css

```diff
@@ -1,6 +1,6 @@
 .resource-section {
-  background: #b6b6c2;
-  padding: 0 30px;
+  display: flex;
+  background-color: #e7e7e7;
```

Comment:
I'd suggest you use a colour like whitesmoke (f5f5f5) for the background colour, as it matches the colour of the about page and creates a bit of a difference between the background and the form container

Reply:
@vrushti-mody Requested changes have been made. I am adding ss of how the page looks like. Doubt: Should I use git ammend to push the changes to the currrent pr or create a new one?
![Screenshot 2021-01-07 105447](https://user-images.githubusercontent.com/58587256/103854575-cd266100-50d6-11eb-98b9-b0057b51ef40.png)
![Screenshot (5)](https://user-images.githubusercontent.com/58587256/103854829-5f2e6980-50d7-11eb-9d09-e4759781487c.png)

Reply:
@Utkarshn10 you just need to add, commit and push by being on the same branch through which you have created this PR. You don't need to create a new PR.

On small devices, either increase the width of the input fields or make those fields align horizontally centered

Reply:
@Kajol-Kumari  Is this good?
![1](https://user-images.githubusercontent.com/58587256/103865149-be957500-50e9-11eb-9c83-21ba4f179309.png)

Reply:
Could you make all the input fields to 100% length instead of 50%?

Reply:
@vrushti-mody I tried it, but since I have added padding to the right of name and phone no. column there is a patch left when the width is 100%. Is there a better way to do it.

Reply:
Try aligning the input to center and messing around with the width. I guess 94-96% would help reduce the patch

Reply:
Could you commit your changes? I'll try to make a few changes locally and get back to you

Reply:
@vrushti-mody I reduced the padding.
![Screenshot (6)](https://user-images.githubusercontent.com/58587256/103872643-b3941200-50f4-11eb-8516-48b4ab7b4c87.png)
![Screenshot (7)](https://user-images.githubusercontent.com/58587256/103872652-ac6d0400-50f4-11eb-9849-bdb4f00a6883.png)